reswift-jobs/reswift-jobs/Model/Employee.swift

```diff
@@ -9,6 +9,8 @@
 import Foundation
 
 struct Employee {
+    static var empty: Employee { return Employee(name: "New Employee", skills:"(skills}") }
```

Comment:
in `skills:"(skills}"`, did you mean to use `(` and `}` together? Looks imbalanced.